mmpose/models/detectors/top_down.py

```diff
@@ -212,6 +224,9 @@ def forward_test(self, img, img_metas, **kwargs):
             output = (output + output_flipped) * 0.5
 
         output_heatmap = output.detach().cpu().numpy()
+        if return_heatmap:
+            self.output_heatmap = output_heatmap
```

Comment:
saving the output heatmap in the class will take some memory even after the function call, right?

Reply:
so i'm in favor of the old way of `return all_preds, all_boxes, image_path, output_heatmap`

you can set `output_heatmap=None` if return_heatmap is false

Reply:
Yeah, agreed.